ext/queue_tasks/managed/Queue.mgd.php

```diff
@@ -0,0 +1,20 @@
+<?php
+return [
+  [
+    'module' => 'queue_tasks',
+    'name' => 'batch_merge',
+    'entity' => 'Queue',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'batch_merge',
+        'type' => 'Sql',
+        'runner' => 'task',
```

Comment:
Clarification -- Setting this `runner=>task` implies that the queue contains `CRM_Queue_Task` records. There is an existing handler (`CRM_Queue_BAO_Queue::hook_civicrm_queueRun_task`) which tries to execute `task` items.

To create a new kind of record (e.g. basic array), in this context, it would look like:

1. Declare the queue with `'runner'=>'batch_merge'`
2. In the handler, subscribe to `hook_civicrm_queueRun_batch_merge`

Reply:
@totten your PR that I just merged changed that though....

Reply:
Ah, sure. The implemntation/listener for `hook_civicrm_queueRun_task` moved from `CRM_Queue_BAO_Queue` to `CRM_Queue_TaskHandler`. In either case, the `task` is already assigned the meaning of "Handle `CRM_Queue_Task`."

Perhaps https://github.com/civicrm/civicrm-core/pull/32233 would make it easier to skim+recognize the connection.